app/helpers/nav_helper.rb

```diff
@@ -6,14 +6,22 @@ def navbar_image_link
     end
   end
 
-  def locale_switcher_links
-    locale_links = []
+  def locale_switcher_buttons
+    locale_links = ['<ul class="navbar-nav">']
     (I18n.available_locales - [I18n.locale]).each do |locale|
-      subdomain = locale.to_s == 'en' ? '' : locale
-      locale_name = I18n.t('name', locale: locale)
-      locale_links << link_to(locale_name, url_for(subdomain: subdomain, only_path: false), class: 'nav-item pl-3')
+      locale_links << '<li class="nav-item pl-3 pr-3 nav-lozenge nav-lozenge-little-padding">' +
+                      link_to(locale_name_for(locale), url_for(subdomain: subdomain_for(locale), only_path: false)) + '</li>'
```

Comment:
This won't will work on the test server, where the domains we're switching between are `test.` and `test-cy.` the code is assuming the switching is between the bare domain and a `cy` subdomain.

We have environment variables for `APPLICATION_HOST` (`test.energysparks.uk`) and `WELSH_APPLICATION_HOST` (`test-cy.energysparks.uk`) which are being used elsewhere. so I'm wondering if it would be better to use those. E.g. if locale is cy use `WELSH_APPLICATION` as the domain, otherwise default to current domain.